src/Events/Events.js

```diff
@@ -307,8 +308,8 @@ class Events extends PureComponent {
                 )}
                 {eventsInSection.map((item) => (
                   <Event
-                    key={item.data.id}
-                    event={item.data}
+                    key={item.ref.id}
+                    event={item.ref}
```

Comment:
> why we don't apply it at this time?

@hoangnm because of the prop `position={item.style}`.
The object `item.style` is re-created each time `processEvents(...)` is called again, even in some cases that the values `top`, `left`, `height` and `width` are all the same

Improving the `position` prop affects other parts of the code, I prefer addressing it in a separate PR

Reply:
thank @pdpino , got it, I forgot that part.